packages/circuit-ui/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { HTMLAttributes } from 'react';
+import { css } from '@emotion/core';
+import isPropValid from '@emotion/is-prop-valid';
+
+import styled, { StyleProps } from '../../styles/styled';
+
+export interface AvatarProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, 'size'> {
+  /**
+   * Image to render
+   */
+  imageUrl?: string;
+  /**
+   * Shape of the Avatar
+   */
+  variant?: 'square' | 'round';
```

Comment:
We're trying to use more semantic names in the design system that make it clear when a certain variant should be used. Giao-Chung used "profile" and "business" in Figma, though I'd personally prefer "person" and "business" (we should keep it aligned between platforms though).

Reply:
I also prefer `person` and `business`, alternatively I've seen `object` and `identity` that could also work. I'll go for person/business for now and we can aways update these to align with the mobile platforms (I've intentionally not given too much thought to prop names so far, wanted to double check with the team)